plugin/server/controllers/provider.js

```diff
@@ -4,6 +4,7 @@ const { getService } = require('../utils/get-service')
 
 module.exports = () => ({
   async usage(ctx) {
-    ctx.body = await getService('provider').usage()
+    const data = await getService('provider').usage()
```

Comment:
Maybe safety check here to not error when the usage function is not defined?

Reply:
I agree but I think the right place would be in `plugin/server/services/provider.js`